File: app/src/main/java/com/carpinelli/testeandroid/model/invest/TwelveMonths.java
package com.carpinelli.testeandroid.model.invest;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

public class TwelveMonths implements Serializable {

    private double fund;
    @JsonProperty("CDI")
    private double CDI;

    public double getFund() {
        return fund;
    }

    public void setFund(double fund) {
        this.fund = fund;
    }

    public double getCDI() {
        return CDI;
    }

    public void setCDI(double CDI) {
        this.CDI = CDI;
    }
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/TwelveMonths.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class TwelveMonths: Serializable {

    @SerializedName("fund")
    var fund: Double? = null
    @SerializedName("CDI")
    var cdi: Double? = null

    override fun toString(): String {
        return "TwelveMonths(fund=$fund, cdi=$cdi)"
    }
}